package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
why do we need this lockfile at the top level?

Reply:
That is indeed a mistake! Thank you :)